purge-cluster.yml

```diff
@@ -237,7 +237,8 @@
     shell: ceph-disk zap "{{ item }}"
     with_items: devices
     when:
-     osd_group_name in group_names
+     osd_group_name in group_names and
+     not osd_directory
```

Comment:
Since we don't depend on ceph-common we can not evaluate `osd_directory` since it might not be defined...